weblate/addons/tasks.py

```diff
@@ -78,7 +78,7 @@ def cdn_parse_html(files: str, selector: str, component_id: int) -> None:
 )
 def language_consistency(addon_id: int, language_ids: list[int]) -> None:
     addon = Addon.objects.get(pk=addon_id)
-    project = addon.component.project
+    project = addon.project
```

Comment:
This now needs to get project_id as a parameter, as it can no longer can be inferred from the add-on for site-wide add-ons.